tests/python/pants_test/engine/test_scheduler.py

```diff
@@ -221,22 +221,19 @@ def test_transitive_params(self):
 
     # Test that an inner Get in transitive_coroutine_rule() is able to resolve B from C due to the
     # existence of transitive_b_c().
-    result_d = self.request_single_product(D, Params(c))
-    # We don't need the inner B objects to be the same, and we know the arguments are type-checked,
-    # we're just testing transitively resolving products in this file.
-    self.assertTrue(isinstance(result_d, D))
+    with self.assertDoesNotRaise():
+      _ = self.request_single_product(D, Params(c))
 
   @contextmanager
   def _assert_execution_error(self, expected_msg):
     with assert_execution_error(self, expected_msg):
       yield
 
   def test_union_rules(self):
-    a = self.request_single_product(A, Params(UnionWrapper(UnionA())))
-    # TODO: figure out what to assert here!
-    self.assertTrue(isinstance(a, A))
-    a = self.request_single_product(A, Params(UnionWrapper(UnionB())))
-    self.assertTrue(isinstance(a, A))
+    with self.assertDoesNotRaise():
+      _ = self.request_single_product(A, Params(UnionWrapper(UnionA())))
+    with self.assertDoesNotRaise():
+      _ = self.request_single_product(A, Params(UnionWrapper(UnionB())))
```

Comment:
Ditto on removing `_ =`

Reply:
I'm honestly a fan of the `_ =`, it says "this has a result, but I choose to ignore it".